internal/batch/batch_manager.go

```diff
@@ -51,7 +51,7 @@ func NewBatchManager(ctx context.Context, ni sysmessaging.LocalNodeInfo, di data
 		startupOffsetRetryAttempts: config.GetInt(config.OrchestratorStartupAttempts),
 		dispatchers:                make(map[fftypes.MessageType]*dispatcher),
 		shoulderTap:                make(chan bool, 1),
-		newMessages:                make(chan int64, readPageSize),
```

Comment:
Seemed arbitrary that we had a page size here, as we have a dedicated channel reading from it